heron/healthmgr/src/java/com/twitter/heron/healthmgr/diagnosers/SlowInstanceDiagnoser.java

```diff
@@ -45,6 +45,9 @@ public Diagnosis diagnose(List<Symptom> symptoms) {
         || !processingRateSkewComponents.isEmpty()) {
       // Since there is no back pressure or disparate wait count or similar
       // execution count, no action is needed
+      LOG.info("no back pressure " + bpSymptoms.isEmpty());
+      LOG.info("no disparate wait count " + waitQDisparityComponents.isEmpty());
+      LOG.info("no similar execution count " + !processingRateSkewComponents.isEmpty());
```

Comment:
These logs seem noisy? Diagnosers will be invoked periodically/frequently and I am not sure how these logs will help the user debug an issue.

Reply:
what we observed is the topology is in backpressure state, but the detector failed to catch
```
[2017-10-03 21:00:26 +0000] [WARNING] com.twitter.heron.healthmgr.policy.AutoRestartBackpressureContainerPolicy: Unknown diagnoses. None resolver selected. 
```
I think these logs helps to find out why the detector failed to detect backpressure